apps/web/pages/v2/bookings/[status].tsx

```diff
@@ -84,15 +84,15 @@ export default function Bookings() {
       heading={t("bookings")}
       subtitle={t("bookings_description")}
       customLoader={<SkeletonLoader />}>
-      <WipeMyCalActionButton bookingStatus={status} bookingsEmpty={isEmpty} />
       <div className="flex w-full flex-1 flex-col">
         {query.status === "error" && (
           <Alert severity="error" title={t("something_went_wrong")} message={query.error.message} />
         )}
         {(query.status === "loading" || query.status === "idle") && <SkeletonLoader />}
         {query.status === "success" && !isEmpty && (
           <div className="pt-2 xl:mx-6 xl:pt-0">
-            <p className="pb-3 text-xs font-medium leading-4 text-gray-500">Today</p>
+            <WipeMyCalActionButton bookingStatus={status} bookingsEmpty={isEmpty} />
```

Comment:
Wouldn't be better in the same line as "Today" and at the right?

<img width="1482" alt="image" src="https://user-images.githubusercontent.com/467258/188174041-468df93a-7bbd-48ff-a317-aaf626661f73.png">

Reply:
yeah but doesnt make a huge difference since not many people use wipe my cal yet